<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EvaluateTab</class>
 <widget class="QWidget" name="EvaluateTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>588</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Courier New</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="TextEdit" name="EvalExp">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Expression</string>
         </property>
         <property name="tabChangesFocus">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>Expression</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="EvalVarSub">
         <property name="toolTip">
          <string>Type in variable and what to subtitute variable with separated by a ':'. Example: x: pi/2. Don't forget to separate varaible and value with space</string>
         </property>
         <property name="placeholderText">
          <string>Variable Substitution</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="EvalPrev">
         <property name="toolTip">
          <string>Preview the expression</string>
         </property>
         <property name="text">
          <string>Preview</string>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="EvalCalc">
         <property name="toolTip">
          <string>Simplify the expression - Shift + Enter</string>
         </property>
         <property name="text">
          <string>Calculate</string>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QTextBrowser" name="EvalOut">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Simplified Answer</string>
       </property>
       <property name="lineWrapMode">
        <enum>QTextEdit::NoWrap</enum>
       </property>
       <property name="placeholderText">
        <string>Output</string>
       </property>
      </widget>
      <widget class="QTextBrowser" name="EvalApprox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Approximate answer of calculation</string>
       </property>
       <property name="placeholderText">
        <string>Approximate Answer</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TextEdit</class>
   <extends>QPlainTextEdit</extends>
   <header>caspy3/qt_assets/highlight.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
